requirements.txt

```diff
@@ -7,7 +7,7 @@ cwltool>=1.0.20191225192155
 docutils
 ephemeris>=0.10.3
 galaxy-containers
-galaxy-tool-util>=20.9.0.dev2
+galaxy-tool-util>=21.1.0.dev4
```

Comment:
This could be an issue when updating the conda package for planemo, since this is a pre-release and there is no 21.1.x proper release.

Reply:
We should be able to get a conda package for this as well - if needed.

Reply:
True, but I didn't have the impression that having a up-to-date conda package was a high priority (the currently specified version is also a pre-release).

Reply:
We can also wait for the official 21.01, it doesn't really bother me.

Reply:
> the currently specified version is also a pre-release

Yes, but there is 20.9.x non pre-release package to satisfy the `>=` requirement. Any way it's not an issue until this ends in a Planemo release and it's good to have this merged at some point.

Reply:
if you have concerns, we could also create a dev branch and merge those PRs into the dev branch. I do think its worth to start working on new features during Galaxy release phase.